code/modules/jobs/job_types/medical.dm

```diff
@@ -48,6 +48,14 @@ Chief Medical Officer
 
 	chameleon_extras = list(/obj/item/gun/syringe, /obj/item/stamp/cmo)
 
+/datum/outfit/job/cmo

```

Comment:
Overwrites /datum/outfit/job/cmo
change to /datum/outfit/job/cmo/hardsuit

Reply:
Darn, don't drink and PR kids